framework/JobHandler.py

```diff
@@ -534,6 +551,7 @@ def getFinished(self, removeFinished=True, jobIdentifier = '', uniqueHandler = "
         ExternalRunner objects) (if jobIdentifier is None), else the finished
```

Comment:
The finished list is now a list of lists, so the docstring needs to be updated.

Reply:
From what I see this is still a list

Reply:
It is a list of lists, as in ```[run, run, [run, run, run], run]``` etc. This could be changed by switching from append to extend when adding the batched lists (as I have commented before.)

Reply:
Okay, as we discussed in the meeting, we will switch it back to a plain list, so the documentation does not need to be changed.